adoc/admin-updates.adoc

```diff
@@ -241,6 +241,47 @@ kubectl -n kube-system annotate ds kured weave.works/kured-node-lock='{"nodeID":
 
 This command modifies an annotation (`annotate`) on the daemonset (`ds`) named `kured`.
 
+When automatic reboots are disabled, you will have to manage reboots yourself.
+In order to do this, you will have to follow some steps whenever you want to issue a reboot marker for a node.
+First of all, you will have to `cordon` and link:{kubedoc}tasks/administer-cluster/safely-drain-node/[`drain`] the node:
+
+----
+kubectl cordon <NODE_ID>
+kubectl drain --force=true \
+  --ignore-daemonsets=true \
```

Comment:
@r0ckarong you need to update this part, then. There is a list of 4 tips below but only three are listed here.